test/unit/presenters/publishing_api_presenters/case_study_test.rb

```diff
@@ -80,6 +80,21 @@ def present(edition)
     assert_equal expected_hash, presented_hash[:details][:image]
   end
 
+  test "handles the case study image caption being blank" do
```

Comment:
nitpick: it would be better to describe _how_ the presenter handles the case study image caption being blank. maybe change the title to: "returns case study image caption as nil when it is blank".